docs/book/mlops-stacks/orchestrators/kubeflow.md

```diff
@@ -130,6 +130,17 @@ When using the Kubeflow orchestrator locally, you'll additionally need
 cluster.
 * A [local container registry](../container-registries/default.md) as part of your stack.
 
+{% hint style="warning" %}
+If you're using the Kubeflow orchestrator locally in combination with a
+[local metadata store](../metadata-stores/sqlite.md), you need to make sure the SQLite
+database is writable from inside the Kubeflow Pods. To do so, first figure our where
+your global config directory is using [this guide](../../resources/global-config.md#the-global-config)
+and then run:
+```shell
+chmod -R a+rw <GLOBAL_CONFIG_DIR>/local_stores
```

Comment:
Doing this on the local filesystem basically gives everyone on the machine read/write access to all these files in your home directory (on Linux at least), which includes your local secrets manager stores.

Reply:
The user in slack still ran into issues, even with the UID/GID commands in the image. It might not be related to Kubeflow after all, as there seems to be some SQLite issues when the user doesn't have write access to the entire directory containing the database.
I guess this call could also be limited to exactly this directory by doing `chmod -R a+rw <GLOBAL_CONFIG_DIR>/local_stores/<UUID>`, which only contains the sqlite database. However I also don't have a problem not adding that at all if you think it's not a common error